src/routes/docs/advanced/self-hosting/functions/+page.markdoc

```diff
@@ -129,4 +129,99 @@ You can find a full list of supported runtimes [here](/docs/products/functions/r
 
 You can also configure the maximum timeout that can be set on individual Appwrite Functions. The maximum configurable timeout can be increased by changing the `_APP_FUNCTIONS_TIMEOUT` environment variable. This environment variable changes the configurable maximum but does not alter existing configurations of individual functions.
 
-{% partial file="update-variables.md" /%}
\ No newline at end of file
+{% partial file="update-variables.md" /%}
+
+# SSL certificates for Function domains {% #ssl-certificates %}
+
+Appwrite does not handle certificates for function domains (e.g. `6772722a00331315adc3.functions.appwrite.myapp.com`)
+out of the box, since they require wildcard certificates.
+There are two ways to handle certificate generation.
+
+## Manual certificate generation
+
+The simplest way to generate certificates for function domains is to use the Appwrite SSL command.
+
+```bash
+docker compose exec appwrite ssl --domain="6772722a00331315adc3.appwrite.myapp.com"
+```
+
+The certificate should be generated within a few seconds.
+If you encounter any issues, you can check the certificate worker logs.
+
+```bash
+docker compose logs appwrite-worker-certificates
+```
+
+Note that you'll need to run this command for each function domain, and repeat it every time you create a new function.
+If you have many functions or frequently create new ones, consider using the automated certificate generation method below.
+
+## Automated certificate generation
+
+For automated certificate generation, Appwrite uses Traefik's DNS Challenge feature.
+This is required for wildcard certificates (like `*.functions.appwrite.myapp.com`)
+because Let's Encrypt uses the DNS-01 challenge to validate wildcard domain ownership.
+
+### Using DNS challenge with DigitalOcean
```

Comment:
Any reason why we chose to provide the steps for DigitalOcean over others or non at all?

Reply:
Matej suggested we add an example for Digitalocean (as we have experience with them) and then link to Traefik docs for the others.

Should I add for others, or just link to them all?